subsys/bluetooth/mesh/shell.c

```diff
@@ -2257,11 +2228,28 @@ static int cmd_attention_set_unack(const struct shell *shell, size_t argc,
 
 	return 0;
 }
+#endif /* CONFIG_BT_MESH_HEALTH_CLI */
+
+static struct bt_mesh_elem *primary_element(void)
+{
+	if (bt_mesh_comp_get() != NULL) {
+		return &(bt_mesh_comp_get()->elem[0]);
+	}
```

Comment:
Since you need to call `bt_mesh_comp_get()` twice, I think it'd improve readability if you just stored it in a variable, i.e.

```
struct bt_mesh_comp *comp = bt_mesh_comp_get();

if (comp) {
        return &comp->elem[0];
}

return NULL;
```